extensions/arcade/sawtooth_battleship/battleship_cli.py

```diff
@@ -228,8 +228,8 @@ def do_init(args, config):
     save_config(config)
 
     wif_filename = config.get('DEFAULT', 'key_file')
-    if wif_filename.endswith(".wif"):
-        addr_filename = wif_filename[0:-len(".wif")] + ".addr"
+    if wif_filename.endswith(".priv"):
+        addr_filename = wif_filename[0:-len(".priv")] + ".addr"
```

Comment:
Probably for another PR, but this address can be generated from the contents of the `.pub` file